src/lib.rs

```diff
@@ -240,15 +240,15 @@
 //! # `read` / `modify` / `write` API
 //!
 //! Each register in the register block, e.g. the `cr1` field in the `I2C` struct, exposes a
-//! combination of the `read`, `modify` and `write` methods. Which methods exposes each register
+//! combination of the `read`, `modify`, and `write` methods. Which method exposes each register
```

Comment:
Looks good to me except these strange commas before `and`.

Reply:
Those are Oxford commas.